falcon/asgi/request.py

```diff
@@ -347,7 +349,7 @@ class Request(falcon.request.Request):
         'scope',
     ]
 
-    def __init__(self, scope, receive, options=None):
+    def __init__(self, scope, receive, first_event=None, options=None):
```

Comment:
`Request` classes are at the core of the Falcon req-resp cycle, we would need to microbenchmark first.

Reply:
I added this as an experiment...probably should benchmark to see if there is any impact. But I like the idea of using this to avoid accidentally passing values into the wrong args as has happened in the past when adding or rearranging kwargs.

Reply:
I just ran several tests with an empty `pass` function and it turns out that using `*` costs about 5-10ms on my MBP. I think I will go ahead and remove it just be safe. It may still be a useful refactoring tool (i.e., add it in when changing the args list to catch any mistakes, then remove once everything looks good and tests are green).